hhub/views.py

```diff
@@ -1,10 +1,29 @@
-from django.http import HttpResponse, JsonResponse
+from django.http import HttpResponse, JsonResponse, FileResponse
 
 from hhub.models import Entry
 from hhub.serializers import EntrySerializer
 import json
 from django.core.paginator import Paginator, EmptyPage, PageNotAnInteger
 
+# given a filename, returns that file
+# /entry/<entry-slug>/<filename>
+def entry_download_file(request, pk, filename):
```

Comment:
Why are you replacing the route serving the game manifest with one serving every file?

Reply:
Not sure if I got it;
What I tried to achieve, in poor words is: "hey, if I ask for file named 2048.gb, gimme that file if exists in <entry-slug> and it is called <filename>"